release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+
+[id="deprecated-and-removed-features_{context}"]
+== Deprecated and removed features
+
+Some features available in earlier releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+For the most recent list of major functionality deprecated and removed, see the following table.
+Additional removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+* GA: General Availability
+* TP: Technology Preview
+* DEP: Deprecated
+* REM: Removed
+* NA: Not applicable
+
+.Deprecated and removed features tracker
+[cols="4,1,1,1",options="header"]
+|===
+|Feature |{product-title-short} 4.7 |{product-title-short} 4.8 |{product-title-short} 4.9
+
+a|Admission controller configuration parameters:
+
+* `admissionControl.contactImageScanners` 
+* `admissionControl.dynamic.enforceOnCreates`
+* `admissionControl.dynamic.enforceOnUpdates`
+* user configuration ability of `admissionControl.dynamic.scanInline` 
+* user configuration ability of `admissionControl.dynamic.timeout` 
+* `admissionControl.listenOnCreates`
+* `admissionControl.listenOnEvents`
+* `admissionControl.listenOnUpdates`
+* `admissionControl.timeoutSeconds`
+
+|GA
+|GA
+|DEP
+
+|API token authentication for {cloud-redhat-com}^[1]^
+|DEP
+|DEP
+|DEP
+
+|Collections hierarchical implementation^[2]^
+|GA
+|GA
+|DEP 
+
+|Compliance dashboard
+|NA
+|DEP
+|DEP
+
+|`definitions.stackrox.io`
+|DEP
+|DEP
+|DEP
+
+|*Export Download Evidence as CSV* button^[3]^
+|GA
+|GA
+|REM
+
+|Google Container Registry integration^[4]^
+|DEP
+|DEP
+|DEP
+
+|GraphQL endpoints^[5]^
+|GA
+|GA
+|DEP
+
+|GraphQL image -> scan -> components and image -> scan -> components -> vulns queries^[6]^
+|GA
+|GA
+|REM
+
+|Kernel support packages and driver download functionality^[7]^
+|DEP
+|DEP
+|DEP
+
+|Manifest installation method, also called the `roxctl` CLI installation method^[8]^
+|GA
+|GA
+|DEP
+
+|Reporting of Istio vulnerabilities
+|DEP
+|DEP
+|DEP
+
+a|`roxctl` admission controller parameters: 
+
+* `--admission-controller-enforce-on-creates`
+* `--admission-controller-enforce-on-updates`
+* `--admission-controller-listen-on-creates`
+* `--admission-controller-listen-on-updates`
+* `--admission-controller-listen-on-events`
+* `--admission-controller-timeout`
+|GA
+|GA
+|DEP
+
+|StackRox Scanner
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/suppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/unsuppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/suppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/unsuppress` APIs^[9,10]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/summary/counts` endpoint
+|DEP
+|DEP
+|REM
+
+|Vulnerability Management (1.0) menu item^[11]^
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Report Creator permission
+|DEP
+|DEP
+|DEP
+
+|===
+
+[.small]
+--
```

Comment:
General comment for these statements: 

- I see both "is deprecated" and "is now deprecated" forms, so remove the "now" 
- Is "is deprecated" the general standard for these statements? It's a bit jarring to go from "is deprecated" in bullet 1, then "has been removed" in bullet 2, etc.